src/views/ReturnDetails.vue

```diff
@@ -3,14 +3,21 @@
     <ion-header :translucent="true">
       <ion-toolbar>
         <ion-back-button default-href="/" slot="start"></ion-back-button>
-        <ion-title>{{ $t("Return Details") }}</ion-title>
+        <ion-title>{{ current.externalId }}</ion-title>
       </ion-toolbar>
     </ion-header>
 
     <ion-content>
       <main>
         <ion-item lines="none">
-          <h1>{{ $t("Shipment ID") }}: {{ current.shipmentId }}</h1>
+          <ion-list class="ion-text-center">
+            <ion-title>{{ $t("Order: <order name>") }}</ion-title>
```

Comment:
If we are not having any specific property like order name then for now, comment the code and add a TODO.

Reply:
Sure sir